src/constants/addresses.ts

```diff
@@ -16,12 +16,13 @@ export const V2_ROUTER_ADDRESS: AddressMap = constructSameAddressMap(
   '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D',
   false
 )
+
+// ordered list of addresses where 0 index is current address
 export const GOVERNANCE_ADDRESSES: AddressMap[] = [
-  constructSameAddressMap('0x5e4be8Bc9637f0EAA1A755019e06A68ce081D58F', false),
-  {
-    [SupportedChainId.MAINNET]: '0xC4e172459f1E7939D522503B81AFAaC1014CE6F6',
-  },
-]
+  '0xC4e172459f1E7939D522503B81AFAaC1014CE6F6',
+  '0x5e4be8Bc9637f0EAA1A755019e06A68ce081D58F',
+].map((address) => constructSameAddressMap(address, false))
```

Comment:
we're going to have to drop testnet support for upgraded governor contracts, only the first governor is on all networks, `0xC4e172459f1E7939D522503B81AFAaC1014CE6F6` is mainnet-only